pkgs/development/python-modules/deeplabcut/default.nix

```diff
@@ -0,0 +1,111 @@
+{ lib, pkgs, buildPythonPackage, fetchPypi, python3Packages
+, cudaSupport ? false, nvidia_x11 ? null, cudatoolkit ? null, cudnn ? null
+, python3
+, fetchzip
+, my_opencv
+}:
+
+assert cudaSupport -> cudatoolkit != null
+                   && cudnn != null
+                   && nvidia_x11 != null;
+
+let
+my_tf = with pkgs; python3Packages.tensorflow.override {
+      cudaSupport = true; nvidia_x11 = linuxPackages.nvidia_x11;
+      cudatoolkit = cudatoolkit; cudnn = cudnn;
+};
+in
+buildPythonPackage rec {
+  pname = "deeplabcut";
+  version = "2.0.5";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "19gxd1nij39lbvlpsmx0ibnfcadxbk5n5a3qsyrgv68r6k6i8rk8";
+  };
+
+  resnet50 = fetchzip {
+    url = "http://download.tensorflow.org/models/resnet_v1_50_2016_08_28.tar.gz";
+    sha256 = "0n9k24ipgivvvd42h2bn26n4yjab0103ikjwhgcg668vbd5mv587";
+ };
+  resnet101 = fetchzip {
+    url = "http://download.tensorflow.org/models/resnet_v1_101_2016_08_28.tar.gz";
+    sha256 = "0h7b963px3xma936zysj54c8l930h3jyss7jv0gzbxalvxzq459x";
+ };
+
+  nativeBuildInputs = with pkgs; with python3Packages; [ setuptools_scm curl gzip gnutar];
+  /* buildInputs = with pkgs; [ tk tcl]; */
+  propagatedBuildInputs = with pkgs; with python3Packages; [ setuptools scipy certifi
+      chardet
+      click
+      easydict
+      h5py
+      imageio
+      ipython
+      /* ipython-genutils */
+      matplotlib
+      moviepy
+      numpy
+      my_opencv
```

Comment:
The more normal thing to do with things like this is to take an argument expecting a regular package (`opencv`) and perform the `enableFfmpeg` override inside the package itself, rather than funny-named arguments.

Reply:
Makes sense, thx!